src/assets/locales/resources_en_US.json

```diff
@@ -11,6 +11,14 @@
     "message": "Toggle sidebar",
     "_description": "Button text to show or hide sidebar menu"
   },
+  "stache_copy_to_clipboard_button_title": {
+    "message": "Copy to clipboard",
+    "_description": "Button title to for the copy to clipboard button"
+  },
+  "stache_copy_to_clipboard": {
```

Comment:
I would suffix this with `_button`. Right now it's too generic and isn't immediately obvious (when viewed in the templates) what its purpose is.